package.json

```diff
@@ -16,45 +16,40 @@
   },
   "main": "lib/fiware-pep-steelskin",
   "engines": {
-    "node": ">=4.8.4"
+    "node": ">=8.12.0"
   },
   "scripts": {
-    "test": "grunt"
+    "clean": "rm -rf package-lock.json && rm -rf node_modules && rm -rf coverage",
+    "test": "mocha --recursive 'test/unit/**/*.js' --reporter spec --timeout 3000 --ui bdd --exit",
+    "test:tools": "mocha --recursive 'test/tools/**/*.js' --reporter spec --timeout 3000 --ui bdd --exit",
```

Comment:
Gruntfile used for unit test:

```
            test: {
                src: ['test/unit/**/*.js', 'test/tools/**/*.js'],
                options: {
                    jshintrc: 'test/.jshintrc'
                }
},
```

So I have created one script (the default one) for test/unit and other (`test:tools`) for test/tools.

However, having said that, not sure if `test:tools` is really doing anything. I don't know what is the purpose of the code under `test/tools`. Maybe the  `test:tools` script can be removed? Maybe even the code under test/tools can be removed?

Reply:
`npm test` result:

```
257 passing (20s)
```

Reply:
`npm run test:tools` result:

```
0 passing (6ms)
```

Reply:
Removing tools code, then run tests:

```
$ rm test/tools/*
$ npm test

> fiware-pep-steelskin@1.4.0-next test /data/src/fiware-pep-steelskin
> mocha --recursive 'test/unit/**/*.js' --reporter spec --timeout 3000 --ui bdd --exit

module.js:550
    throw err;
    ^
```

Reply:
In sum, it seems code in test/tools is needed by test/units (so it cannot be removed) but there isn't any runable test un test/tools itself (so the tests:tools npm script can be removed).

Fixed in 9bf9922